src/dsp_tools/commands/xmllib/models/root.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from dataclasses import field
+from pathlib import Path
+
+from lxml import etree
+
+from dsp_tools.commands.excel2xml import append_permissions
+from dsp_tools.commands.excel2xml import write_xml
+from dsp_tools.commands.xmllib.models.resource import Resource
+
+XML_NAMESPACE_MAP = {None: "https://dasch.swiss/schema", "xsi": "http://www.w3.org/2001/XMLSchema-instance"}
+DASCH_SCHEMA = "{https://dasch.swiss/schema}"
+
+
+@dataclass
+class XMLRoot:
+    shortcode: str
+    default_ontology: str
+    resource_collection: list[Resource] = field(default_factory=list)
```

Comment:
the term `resource_collection` is a leftover of previous discussions, in the current state it no longer makes much sense. In terms of the `XMLRoot`, this should just be `resources`, so I would suggest renaming it to that